orte/mca/schizo/base/schizo_base_stubs.c

```diff
@@ -46,6 +46,11 @@ int orte_schizo_base_parse_env(char *personality,
     int rc;
     orte_schizo_base_active_module_t *mod;
 
+    if (NULL == personality) {
+        opal_output(0, "NULL PERSONALITY");
```

Comment:
Ok, thanks.

General question: It looks like these are assert-like things, right?  I.e., these should never happen / are only there as a failsafe, right?